heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraContext.java

```diff
@@ -14,16 +14,40 @@
 
 package com.twitter.heron.scheduler.aurora;
 
+import java.io.File;
+
 import com.twitter.heron.spi.common.Config;
 import com.twitter.heron.spi.common.Context;
 
 public final class AuroraContext extends Context {
   public static final String JOB_LINK_TEMPLATE = "heron.scheduler.job.link.template";
+  public static final String JOB_TEMPLATE_ENV = "HERON_SCHEDULER_JOB_TEMPLATE";
+  public static final String JOB_TEMPLATE = "heron.scheduler.job.template";
+  public static final String JOB_TEMPLATE_DIR = "heron.scheduler.job.template.dir";
 
   private AuroraContext() {
   }
 
   public static String getJobLinkTemplate(Config config) {
     return config.getStringValue(JOB_LINK_TEMPLATE);
   }
+
+  /**
+   * Get the path to the Aurora Job template to be used by heron
+   * If the environment parameter HERON_SCHEDULER_JON_TEMPLATE is defined then it is used
+   * Otherwise checks the configuration properties heron.scheduler.job.template and
+   * heron.scheduler.job.template.dir for the location of the template.
+   * If heron.scheduler.job.template is not defined then "heron.aurora" is used
+   * If heron.scheduler.job.template.dir is not defined then the cluster specific conf directory
+   * is used.
+   */
+  public static String getHeronAuroraPath(Config config) {
+    String template = System.getenv(JOB_TEMPLATE_ENV);
```

Comment:
Instead of mixing both Heron configs and environment variables can we just use configs? Splitting the logic between configs and env variables makes for an inconsistent experience with hidden features. If we use a config param you can still pass it at the command line to override the default:
```
$ heron help submit | grep config-property
  --config-property (key=value; a config key and its value; default: [])
```